.github/workflows/python-ci-wheel.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       # Build the wheels for Linux, Windows and macOS for Python 3.9
       matrix:
         os: [macos-latest, windows-latest, ubuntu-20.04]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

Comment:
I had problems with "3.10" in here, does it work locally for you?

Reply:
I can merge it to develop-pypi to test.

Reply:
see https://github.com/rettinghaus/verovio/runs/5154734671?check_suite_focus=true

Reply:
Looks like just quoting the value works? https://github.com/luxonis/depthai-python/actions/runs/1335184627/workflow#L111

Reply:
I've added it to just 3.10, and pushed to develop-pypi to see, but I can also quote all the values too

Reply:
Yes, seems like version numbers should be quoted: https://github.com/actions/setup-python/issues/160#issuecomment-724485470 )see also the README examples in that repo)

Reply:
It looks like we're still using a pinned and custom version of cibuildwheel, even though the fix is in the latest version. Should I also update cibuildwheel in the GH action?

(I wonder where this rabbit hole goes...)

Reply:
@musicEnfanthen is the only magician I know who can fix it...

Reply:
@musicEnfanthen do you want to take this task, or should I?

Reply:
@ahankinson I think I figured it out, see #2638